src/main/java/com/blackducksoftware/integration/alert/provider/hub/HubProperties.java

```diff
@@ -52,7 +52,7 @@
 import com.google.gson.Gson;
 
 @Component
-public class GlobalProperties {
+public class HubProperties {
```

Comment:
This should be renamed to BlackDuckProperties and the package should be named blackduck not hub